README.md

```diff
@@ -68,10 +69,11 @@ NOTE: Depending on the version of Charmed Kubeflow you want to test, make sure t
 - Charmed Kubeflow 1.9 -> `track/1.9`
 - Charmed Kubeflow 1.8 -> `track/1.8`
 - Charmed Kubeflow 1.7 -> `track/1.7`
+`main` branch is generally used for testing against the `latest/edge` track of the bundle.   
```

Comment:
We should add a newline before that because it's being rendered on the save line with `1.7`.